bin/set-cumulus-version.sh

```diff
@@ -36,6 +36,6 @@ echo "- ${_}"
 # from the rest of the Cumulus dependencies.
 
 for _filename in package.json scripts/package.json; do
-  perl -i -pe "s/(\@cumulus\/(?!cumulus-message-adapter-js).+\"\s*:\s*)\"([^~]?[0-9]+(?:\.[0-9]+){2})\"/\$1\"${_version}\"/" "${_filename}"
+  perl -i -pe "s/(\@cumulus\/(?!cumulus-message-adapter-js)[^\/]+\"\s*:\s*)\"([^~]?[0-9]+(?:\.[0-9]+){2})\"/\$1\"${_version}\"/" "${_filename}"
```

Comment:
This change was required to prevent this script from erroneously changing version numbers for dependencies in `package.json` that are _not_ Cumulus packages.

For example, without this change, the entries in `package.json` in the `"resolutions"` section would mistakenly be changed when running this script to set the Cumulus version:

```plain
    "@cumulus/**/@aws-sdk/client-s3": "^3.370.0",
    "@cumulus/**/@aws-sdk/client-sts": "^3.370.0",
    "@cumulus/**/@aws-sdk/lib-dynamodb": "^3.370.0",
    "@cumulus/**/@aws-sdk/lib-storage": "^3.370.0",
    "@cumulus/**/@aws-sdk/types": "^3.370.0",
    "@cumulus/**/got": "^11.8.6",
    "@cumulus/**/xml2js": "^0.6.0",
```

Even though those entries start with `"@cumulus"`, they are _not_ Cumulus packages themselves, and thus must _not_ be set to the Cumulus version.

This change prevents them from mistakenly changing.